it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/IOLoadTestBase.java

```diff
@@ -63,7 +63,7 @@ public void tearDownBase() throws IOException {
 
   @Override
   PipelineLauncher launcher() {
-    return DefaultPipelineLauncher.builder().setCredentials(CREDENTIALS).build();
```

Comment:
(mark for my self, not blocking PR) let me check if the setup gcp env step could be removed in github workflow after this change